pkg/utils/perm_windows.go

```diff
@@ -21,10 +27,14 @@ type FileOwner struct {
 
 // CurrentFileOwner returns the executing UID and GID of the current process.
 func CurrentFileOwner() FileOwner {
-	// TODO(blakerouse): Make this return the current user and group on Windows.
+	u, err := user.Current()
+	if err != nil {
+		// should not fail; if it does then there is a big problem
+		panic(err)
```

Comment:
I understand why you used `panic()` here as such an error would not be easily recoverable but I would still prefer to give the chance to the callers to explicitly handle an error condition by returning `error`.
If we want to keep the `panic()` for windows we should at least add this to the documentation of the function for all the OSes (as some people developing on a platform different from windows should be aware that this function can panic on windows)

Reply:
Added comment.